### ISSUE TITLE
Instant Error

### ISSUE THREAD

**Issue by ImInTheICU** [Anarchick/skript-packet#24]
```
[00:55:59 ERROR]: #!#! 
[00:55:59 ERROR]: #!#! [Skript] Severe Error:
[00:55:59 ERROR]: #!#! 
[00:55:59 ERROR]: #!#! Something went horribly wrong with Skript.
[00:55:59 ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[00:55:59 ERROR]: #!#! It looks like you are using some plugin(s) that alter how Skript works (addons).
[00:55:59 ERROR]: #!#! Here is full list of them:
[00:55:59 ERROR]: #!#! skript-reflect v2.3 (https://github.com/TPGamesNL/skript-reflect) Skript-Packet v2.1.1 (www.github.com/Anarchick/skript-packet) 
[00:55:59 ERROR]: #!#! We could not identify which of those are specially related, so this might also be Skript issue.
[00:55:59 ERROR]: #!#! You should try disabling those plugins one by one, trying to find which one causes it.
[00:55:59 ERROR]: #!#! If the error doesn't disappear even after disabling all listed plugins, it is probably Skript issue.
[00:55:59 ERROR]: #!#! In that case, you will be given instruction on how should you report it.
[00:55:59 ERROR]: #!#! On the other hand, if the error disappears when disabling some plugin, report it to author of that plugin.
[00:55:59 ERROR]: #!#! Only if the author tells you to do so, report it to Skript's issue tracker.
[00:55:59 ERROR]: #!#! 
[00:55:59 ERROR]: #!#! Stack trace:
[00:55:59 ERROR]: #!#! java.lang.NoSuchMethodError: 'ch.njol.skript.config.Config ch.njol.skript.lang.parser.ParserInstance.getCurrentScript()'
[00:55:59 ERROR]: #!#!     at skript-reflect.jar//com.btk5h.skriptmirror.util.SkriptUtil.getCurrentScript(SkriptUtil.java:108)
[00:55:59 ERROR]: #!#!     at skript-reflect.jar//com.btk5h.skriptmirror.skript.Types$3.parse(Types.java:60)
[00:55:59 ERROR]: #!#!     at skript-reflect.jar//com.btk5h.skriptmirror.skript.Types$3.parse(Types.java:57)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.registrations.Classes.parseSimple(Classes.java:468)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.registrations.Classes.parse(Classes.java:497)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.lang.SkriptParser.parseSingleExpr(SkriptParser.java:593)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.lang.SkriptParser.parseExpression(SkriptParser.java:760)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:154)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:65)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.patterns.OptionalPatternElement.match(OptionalPatternElement.java:46)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:65)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:65)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.patterns.OptionalPatternElement.match(OptionalPatternElement.java:43)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.patterns.SkriptPattern.match(SkriptPattern.java:58)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1273)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:229)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.lang.SkriptParser.parseStatic(SkriptParser.java:204)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//org.skriptlang.skript.lang.structure.Structure.parse(Structure.java:185)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.ScriptLoader.loadScript(ScriptLoader.java:632)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.ScriptLoader.lambda$loadScripts$3(ScriptLoader.java:501)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.ScriptLoader.lambda$makeFuture$2(ScriptLoader.java:422)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.ScriptLoader.makeFuture(ScriptLoader.java:438)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.ScriptLoader.loadScripts(ScriptLoader.java:500)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.ScriptLoader.loadScripts(ScriptLoader.java:457)
[00:55:59 ERROR]: #!#!     at Skript-2.7.0-beta2.jar//ch.njol.skript.SkriptCommand.onCommand(SkriptCommand.java:141)
[00:55:59 ERROR]: #!#!     at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[00:55:59 ERROR]: #!#!     at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155)
[00:55:59 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_19_R3.CraftServer.dispatchCommand(CraftServer.java:906)
[00:55:59 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_19_R3.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:64)
[00:55:59 ERROR]: #!#!     at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:264)
[00:55:59 ERROR]: #!#!     at net.minecraft.commands.CommandDispatcher.performCommand(CommandDispatcher.java:322)
[00:55:59 ERROR]: #!#!     at net.minecraft.commands.CommandDispatcher.a(CommandDispatcher.java:306)
[00:55:59 ERROR]: #!#!     at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2281)
[00:55:59 ERROR]: #!#!     at net.minecraft.server.network.PlayerConnection.lambda$handleChatCommand$20(PlayerConnection.java:2241)
[00:55:59 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.b(IAsyncTaskHandler.java:59)
[00:55:59 ERROR]: #!#!     at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(Unknown Source)
[00:55:59 ERROR]: #!#!     at net.minecraft.server.TickTask.run(TickTask.java:18)
[00:55:59 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.d(IAsyncTaskHandler.java:153)
[00:55:59 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(IAsyncTaskHandlerReentrant.java:24)
[00:55:59 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1342)
[00:55:59 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197)
[00:55:59 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.x(IAsyncTaskHandler.java:126)
[00:55:59 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.bi(MinecraftServer.java:1319)
[00:55:59 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1312)
[00:55:59 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.c(IAsyncTaskHandler.java:136)
[00:55:59 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.i_(MinecraftServer.java:1290)
[00:55:59 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1178)
[00:55:59 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320)
[00:55:59 ERROR]: #!#!     at java.base/java.lang.Thread.run(Unknown Source)
[00:55:59 ERROR]: #!#! 
[00:55:59 ERROR]: #!#! Version Information:
[00:55:59 ERROR]: #!#!   Skript: 2.7.0-beta2 (latest)
[00:55:59 ERROR]: #!#!     Flavor: skriptlang-github
[00:55:59 ERROR]: #!#!     Date: 23:34:46.821464600
[00:55:59 ERROR]: #!#!   Bukkit: 1.19.4-R0.1-SNAPSHOT
[00:55:59 ERROR]: #!#!   Minecraft: 1.19.4
[00:55:59 ERROR]: #!#!   Java: 18.0.2.1 (OpenJDK 64-Bit Server VM 18.0.2.1+1)
[00:55:59 ERROR]: #!#!   OS: Linux amd64 5.15.0-71-generic
[00:55:59 ERROR]: #!#! 
[00:55:59 ERROR]: #!#! Server platform: Paper
[00:55:59 ERROR]: #!#! 
[00:55:59 ERROR]: #!#! Current node: null
[00:55:59 ERROR]: #!#! Current item: null
[00:55:59 ERROR]: #!#! 
[00:55:59 ERROR]: #!#! Thread: Server thread
[00:55:59 ERROR]: #!#! 
[00:55:59 ERROR]: #!#! Language: english
[00:55:59 ERROR]: #!#! Link parse mode: DISABLED
[00:55:59 ERROR]: #!#! 
[00:55:59 ERROR]: #!#! End of Error.
[00:55:59 ERROR]: #!#! 
```

**Comment by Anarchick** [comment]
Use Skript 2.6.x until a fix is release.

**Comment by ImInTheICU** [comment]
> Use Skript 2.6.x until a fix is release.

Rather not

**Comment by AKtomik** [comment]
*i use last version of skript (2.6.4)  and skript-packet 2.1.1

**Comment by Anarchick** [comment]
Official fix with 2.1.2
